src/common/ceph_context.cc

```diff
@@ -120,9 +120,8 @@ class LockdepObs : public md_config_obs_t {
     }
   }
 
-  const char** get_tracked_conf_keys() const override {
-    static const char *KEYS[] = {"lockdep", NULL};
-    return KEYS;
+  std::vector<std::string> get_tracked_keys() const noexcept override {
+    return std::vector<std::string>{"lockdep"s};
```

Comment:
nit: not sure we really need to repeat the type; perhaps `return {"lockdep"s};` would be enough.

Reply:
You are right. Fixed.